README.md

```diff
@@ -1167,7 +1171,17 @@ Thank you!
 ## Cite
 
 If you use AFLplusplus in scientific work, consider citing [our paper](https://www.usenix.org/conference/woot20/presentation/fioraldi) presented at WOOT'20:
-```
+
+If you use AFLpluplus to compare to your work, please use either `afl-clang-lto`
+or `afl-clang-fast` with `AFL_LLVM_CMPLOG=1` for building targets and
```

Comment:
CMPLOG as default?

Reply:
absolutely. cmplog is the strongest feature in afl++. without it we would not win against libfuzzer and honggfuzz

Reply:
this note is is the wrong place, after a colon.

Reply:
> this note is is the wrong place, after a colon.

I do not understand

Reply:
ah got it.